openmc_models/bcc_phys/passwise/1549days/bcc_model.py

```diff
@@ -0,0 +1,34 @@
+import openmc
+import openmc.deplete
+import numpy as np
+
+dep_file = '../i4-dep-res.h5'
+mat_file = '../i4-mats.xml'
+res = openmc.deplete.Results(dep_file)
+dep_t = res.get_times()
+step_comps = [res.export_to_materials(i, 
+                                      path=mat_file)[0].get_nuclide_densities()
+              for i in range(len(dep_t))]
+
+comp1549days = {}
+for k, v in step_comps[-1].items():
+        comp1549days[k] = {}
+        comp1549days[k] = v[1]
+
+ucot = openmc.Material(name='UCO_TRACKED', material_id=13)
```

Comment:
I think it would be good and fine to make this variable name a little clearer (uco_tracked) or (tracked_uco).